index.html

```diff
@@ -52,7 +52,7 @@
         xref: {
           profile: "web-platform",
           specs: [
-            "permissions",
+            "page-visibility",
```

Comment:
This looks like a bad merge. I explicitly removed this in #333 after the visibility state-related changes.